includes/modules/server/Server.hpp

```diff
@@ -5,27 +5,33 @@
 
 #include "api/User.hpp"
 #include "api/Connection.hpp"
+#include <unistd.h>
 
 class Server : public CommandSender {
 private:
 	string name;
-	map<int, Connection *> connections;
+	vector<Connection *> connections;
+	struct sockaddr_in connectionConfig;
+	const string port;
```

Comment:
Change `port` from `const string` to `const int` type.
```suggestion
	const int port;
```